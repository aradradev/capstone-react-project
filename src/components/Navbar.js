/* eslint-disable import/no-extraneous-dependencies */
import React from 'react';
import { Link } from 'react-router-dom';
import {
  UilArrowRight,
  UilMicrophone,
  UilSetting,
} from '@iconscout/react-unicons';

const Navbar = () => (
  <div className="text-white w-full flex items-center flex-row justify-between p-5 bg-slate-800">
    <Link to="/">
      <UilArrowRight />
    </Link>
    <h1 className=" text-white font-medium text-xl">Countries</h1>
    <div className="flex flex-row items-center px-2 mx-1">
      <UilMicrophone
        size={25}
        className="text-white cursor-pointer transition ease-out hover:scale-125 mx-2"
      />
      <UilSetting
        size={25}
        className="text-white cursor-pointer transition ease-out hover:scale-125"
      />
    </div>
  </div>
);
export default Navbar;
