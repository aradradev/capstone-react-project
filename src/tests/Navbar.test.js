import React from 'react';
import renderer from 'react-test-renderer';
import { MemoryRouter } from 'react-router-dom';
import Navbar from '../components/Navbar';

it('matches snapshot', () => {
  const tree = renderer
    .create(
      <MemoryRouter>
        <Navbar />
      </MemoryRouter>,
    )
    .toJSON();
  expect(tree).toMatchSnapshot();
});
