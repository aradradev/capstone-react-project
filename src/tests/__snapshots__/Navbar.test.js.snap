// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches snapshot 1`] = `
<div
  className="text-white w-full flex items-center flex-row justify-between p-5 bg-slate-800"
>
  <a
    href="/"
    onClick={[Function]}
  >
    <svg
      fill="currentColor"
      height="24"
      viewBox="0 0 24 24"
      width="24"
      xmlns="http://www.w3.org/2000/svg"
    >
      <path
        d="M17.92,11.62a1,1,0,0,0-.21-.33l-5-5a1,1,0,0,0-1.42,1.42L14.59,11H7a1,1,0,0,0,0,2h7.59l-3.3,3.29a1,1,0,0,0,0,1.42,1,1,0,0,0,1.42,0l5-5a1,1,0,0,0,.21-.33A1,1,0,0,0,17.92,11.62Z"
      />
    </svg>
  </a>
  <h1
    className=" text-white font-medium text-xl"
  >
    Countries
  </h1>
  <div
    className="flex flex-row items-center px-2 mx-1"
  >
    <svg
      className="text-white cursor-pointer transition ease-out hover:scale-125 mx-2"
      fill="currentColor"
      height={25}
      viewBox="0 0 24 24"
      width={25}
      xmlns="http://www.w3.org/2000/svg"
    >
      <path
        d="M12,15a4,4,0,0,0,4-4V5A4,4,0,0,0,8,5v6A4,4,0,0,0,12,15ZM10,5a2,2,0,0,1,4,0v6a2,2,0,0,1-4,0Zm10,6a1,1,0,0,0-2,0A6,6,0,0,1,6,11a1,1,0,0,0-2,0,8,8,0,0,0,7,7.93V21H9a1,1,0,0,0,0,2h6a1,1,0,0,0,0-2H13V18.93A8,8,0,0,0,20,11Z"
      />
    </svg>
    <svg
      className="text-white cursor-pointer transition ease-out hover:scale-125"
      fill="currentColor"
      height={25}
      viewBox="0 0 24 24"
      width={25}
      xmlns="http://www.w3.org/2000/svg"
    >
      <path
        d="M19.9 12.66a1 1 0 0 1 0-1.32l1.28-1.44a1 1 0 0 0 .12-1.17l-2-3.46a1 1 0 0 0-1.07-.48l-1.88.38a1 1 0 0 1-1.15-.66l-.61-1.83a1 1 0 0 0-.95-.68h-4a1 1 0 0 0-1 .68l-.56 1.83a1 1 0 0 1-1.15.66L5 4.79a1 1 0 0 0-1 .48L2 8.73a1 1 0 0 0 .1 1.17l1.27 1.44a1 1 0 0 1 0 1.32L2.1 14.1a1 1 0 0 0-.1 1.17l2 3.46a1 1 0 0 0 1.07.48l1.88-.38a1 1 0 0 1 1.15.66l.61 1.83a1 1 0 0 0 1 .68h4a1 1 0 0 0 .95-.68l.61-1.83a1 1 0 0 1 1.15-.66l1.88.38a1 1 0 0 0 1.07-.48l2-3.46a1 1 0 0 0-.12-1.17ZM18.41 14l.8.9-1.28 2.22-1.18-.24a3 3 0 0 0-3.45 2L12.92 20h-2.56L10 18.86a3 3 0 0 0-3.45-2l-1.18.24-1.3-2.21.8-.9a3 3 0 0 0 0-4l-.8-.9 1.28-2.2 1.18.24a3 3 0 0 0 3.45-2L10.36 4h2.56l.38 1.14a3 3 0 0 0 3.45 2l1.18-.24 1.28 2.22-.8.9a3 3 0 0 0 0 3.98Zm-6.77-6a4 4 0 1 0 4 4 4 4 0 0 0-4-4Zm0 6a2 2 0 1 1 2-2 2 2 0 0 1-2 2Z"
      />
    </svg>
  </div>
</div>
`;
